virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -11,21 +11,31 @@ You can also import virtual machines with existing workloads that depend on acce
 
 To attach a virtual machine to an additional network:
 
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
+. Configure a Linux bridge.
+. Configure a `NetworkAttachmentDefinition` for a namespace in the web console or CLI.
+. Attach the virtual machine to the `NetworkAttachmentDefinition` by using either the web console or the CLI:
 ** In the web console, create a NIC for a new or existing virtual machine.
 ** In the CLI, include the network information in the virtual machine configuration.
 
+[NOTE]
+====
+There are multiple methods for configuring a VLAN, including `NetworkAttachmentDefinition` and `NodeNetworkConfigurationPolicy`.
+However, a `NetworkAttachmentDefinition` provides a more efficient and more manageable configuration.
+====
+
 include::modules/virt-networking-glossary.adoc[leveloffset=+1]
 
-[id="virt-creating-network-attachment-definition"]
-== Creating a network attachment definition
```

Comment:
NetworkAttachmentDefinition ?

Reply:
According to our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), if you are referring to the actual instance of the API object, use NetworkAttachmentDefinition . If it is a general or conceptual reference, use "network attachment definition".

In this case, it is a general reference, so we use "network attachment definition".

Reply:
Same here - isn't this a reference to the API object? The header talks about creating an instance of a NetworkAttachmentDefinition, isn't it?

Reply:
Pan's explanation settled this one for me.